File: Source/ses_io/src/wrappers/fortran/wrap_ses_write_word.f90

#undef DEBUG_WRAP

SES_ERROR_FLAG function ses_write_word_f90(the_handle, the_buffer)

   implicit none

   SES_FILE_HANDLE, INTENT(IN) :: the_handle
   SES_WORD_REFERENCE :: the_buffer

   SES_ERROR_FLAG :: return_value, ses_write_word
#ifdef DEBUG_WRAP
   print *, "ses_write_word_f90:  the_handle is ", the_handle
   print *, "ses_write_word_f90:  the_buffer is ", the_buffer
#endif

   return_value = ses_write_word(the_handle, the_buffer)
#ifdef DEBUG_WRAP
   print *, "ses_write_word_f90:  return_value is ", return_value
#endif

   ses_write_word_f90 = return_value

 
end function ses_write_word_f90

File: Source/ses_io/src/wrappers/fortran/wrap_ses_write_word.c


#include "ses_defines.h"
#include "ses_globals.h"
#include "ses_internals.h"
#include "ses_externs.h"

#include "stdio.h"
#include <stdlib.h>

#include <stdio.h>

#undef DEBUG_WRAP

#ifdef LC_UNDER
ses_error_flag ses_write_word_(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef UC_UNDER
ses_error_flag SES_WRITE_word_(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef LC_NOUNDER
ses_error_flag ses_write_word(ses_file_handle* pt_handle, ses_word_reference buffer) {
#else

#ifdef UC_NOUNDER
ses_error_flag SES_WRITE_word(ses_file_handle* pt_handle, ses_word_reference buffer) {
#endif

#endif
#endif
#endif

  ses_error_flag return_value = SES_NO_ERROR;

  ses_file_handle the_handle = *pt_handle;
  ses_word the_word = *buffer;
#ifdef DEBUG_WRAP
  printf("wrap_ses_write_word.c:  the_handle is %d\n", the_handle);
  printf("wrap_ses_write_word.c:  the_word is %e\n", the_word);
#endif

  return_value = ses_write_word(the_handle, the_word);
#ifdef DEBUG_WRAP 
  printf("wrap_ses_write_word.c:  return_value is %d\n", return_value);
#endif
  return return_value;

}

 
